extensions/github/src/githubServer.ts

```diff
@@ -80,7 +80,7 @@ export class GitHubServer {
 		this.updateStatusBarItem(true);
 
 		const state = uuid();
-		const callbackUri = await vscode.env.asExternalUri(vscode.Uri.parse(`${vscode.env.uriScheme}://vscode.github-authentication/did-authenticate`));
+		const callbackUri = await vscode.env.asExternalUri(vscode.Uri.parse(`${vscode.env.uriScheme}://vscode.github/did-authenticate`));
```

Comment:
@RMacfarlane Do we need to update the callback URLs also in the app registrations?

Reply:
Yep, that's right! GitHub actually has control over this so I will reach out on getting this updated